core-web/libs/block-editor/src/lib/extensions/image-tabview-form/components/dot-image-card-list/dot-image-card-list.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+import { DotCMSContentlet } from '@dotcms/dotcms-models';
+import { sanitizeUrl } from '@dotcms/block-editor';
+import { squarePlus } from '../../../../shared/components/suggestions/suggestion-icons';
+
+@Component({
+    selector: 'dot-image-card-list',
+    templateUrl: './dot-image-card-list.component.html',
+    styleUrls: ['./dot-image-card-list.component.scss']
+})
+export class DotImageCardListComponent {
+    @Input() done = false;
```

Comment:
```suggestion
    @Input() preventScroll = false;
```

Or soemthing more explicit?